frontend/packages/client/src/components/Community/CommunityEditorProfile/CommunityEditorProfile.js

```diff
@@ -69,12 +69,18 @@ export default function CommunityEditorProfile({
     // url needs to be sent to backend to update
     if (updates?.logo?.file) {
       setIsUpdatingImage(true);
-      const uploadImg = await uploadFile(data?.logo?.file);
+      const uploadImg = await uploadFile(data?.logo?.file).catch(() => {
+        console.log('Error uploading logo image');
+        return;
+      });
```

Comment:
this is being handled by the onError hook on react-query but we need to catch the error